packages/newspringchurchapp/src/prayer/index.js

```diff
@@ -178,7 +213,23 @@ class PrayerMenu extends PureComponent {
   render() {
     return (
       <>
-        <AddPrayerCardConnected {...this.props} />
+        {this.state.showAddPrayerCard ? (
+          <StyledAddPrayerContainer>
+            <AddPrayerCardConnected {...this.props} />
+          </StyledAddPrayerContainer>
+        ) : null}
+        {!this.state.showAddPrayerCard ? (
+          <StyledContainer>
+            <NSIcon name={'arrow-up'} fill={this.props.tint} size={24} />
```

Comment:
This should be within the button press, too. After testing, trying to click the arrow is a frustrating experience and doesn't cause the view to pan up.